index.html

```diff
@@ -48,237 +48,429 @@
-                      <p class="small">required</p>
-                    </td>
-                    <td>API Credential key associated with client generated <a href="http://api.workpath.co" target="blank">here</a>.</td>
-                  </tr>
-                  <tr>
-                    <td>
-                      <p class="cell"><code class="attribute">redirect_uri</code> <code>string</code></p>
-                      <p class="small">required</p>
-                    </td>
-                    <td>Redirect URL associated with client generated <a href="http://api.workpath.co" target="blank">here</a>.</td>
-                  </tr>
-                  <tr>
-                    <td>
-                      <p class="cell"><code class="attribute">state</code> <code>string</code></p>
-                      <p class="small">required</p>
-                    </td>
-                    <td>Randomly generated string.</td>
-                  </tr>
-                  <tr>
-                    <td>
-                      <p class="cell"><code class="attribute">code_challenge</code> <code>string</code></p>
-                      <p class="small">required</p>
-                    </td>
-                    <td>SHA-256 and base64 encoded version of the code verifier.</td>
-                  </tr>
-                  <tr>
-                    <td>
-                      <p class="cell"><code class="attribute">code_challenge_method</code> <code>string</code></p>
-                      <p class="small">required</p>
-                    </td>
-                    <td>Always use <code>S256</code>.</td>
-                  </tr>
-                  <tr>
-                    <td>
-                      <p class="cell"><code class="attribute">response_type</code> <code>string</code></p>
-                      <p class="small">required</p>
-                    </td>
-                    <td>Always use <code>code</code>.</td>
-                  </tr>
-                </tbody>
-              </table>
+
+            <a id="HelloWorld"></a>
+            <header class="wp-docs-content-tab"></header>
+            <div class="container">
+              <div class="row">
+                <div class="col">
+                  <h1>"Hello World" API Use Example</h1>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col-6">
+                  <p>
+                    In true developer fashion, we have provided a quick "Hello World" tutorial for you to try out! This example below will explain how to authenticate and then create a simple intake using our system.
+                  </p>
+                  <p>The steps we will walk through:</p>
+                  <ol>
+                    <li>Create Your First API Key</li>
+                    <li>Retrieve A Token With Your API Key</li>
+                    <li>Create an Intake</li>
+                    <li>Fetch the Intake</li>
+                  </ol>
```

Comment:
@thetechwitch would it be a lot to link these to their respective sections?

Reply:
Not a lot at all @rstims ! I'll do that real quick!